e2e/plotting.spec.ts

```diff
@@ -223,7 +223,7 @@ test('user can zoom and pan the graph', async ({
   }
 });
 
-test('plots multiple channels on the y axis and displays correct tooltips', async ({
+test('plots multiple channels on the y axis', async ({
```

Comment:
So are we happy with not having a test that the tooltips are working? It's fine not to - as that's functionality from Chart.js that we're not modifying really, and it would now be much harder to test here as we'd need an exact pixel position for the mouse to hover over.